it/v2/utils/CommonGenerators.scala

```diff
@@ -151,18 +156,6 @@ trait CommonGenerators {
     } yield MessageUpdate(status, Some(ObjectStoreURI(uri)))
   }
 
-  implicit val arbitraryObjectStoreURI: Arbitrary[ObjectStoreURI] = Arbitrary {
```

Comment:
This is what ObjectStoreURI should be . so should not be removed. 
Please remove above one
``` implicit lazy val arbitraryObjectStoreURI: Arbitrary[ObjectStoreURI] = Arbitrary {
    Gen.alphaNumStr.map(ObjectStoreURI(_))
  } ```

Reply:
It's still present, it's just before the arbitraryMessageSummary as it now is used in that generator

Reply:
He's saying that this implementation should still be used, not simplified to `Gen.alphaNumStr.map(ObjectStoreURI(_))` (because that's not indicative of what object store URIs would look like)